apcd-cms/src/apps/admin_regis_table/static/admin_regis_table/registrations_table.css

```diff
@@ -1,31 +1,32 @@
 .registration-table {
-    margin-right: auto;
-    text-align: center;
-    column-width: auto;
+    --cell-padding: 5px;
+
+    width: 100%;
+    table-layout: fixed;
 }
 .registration-table th, td {
     text-align: left;
     vertical-align: top;
     letter-spacing: 0px;
     color: #484848;
     opacity: 1;
-    padding-top: 5px;
-    padding-bottom: 5px;
+    padding-top: var(--cell-padding);
+    padding-bottom: var(--cell-padding);
     padding-left: 10px;
     padding-right: 40px;
     border-collapse: collapse;
     border-right: 1px solid #fff;
     border-left: 1px solid #fff;
     border-top: 1px solid #fff;
     border-bottom: 1px solid rgba(0,0,0,1);
-white-space: nowrap;
+}
+.registration-table tr:nth-child(even) {
+    background-color: rgba(198, 198, 198, 0.1);
 }
 .registration-table td {
     border-bottom: 1px solid rgba(0,0,0,0.1);
 }
 .registration-table tr:nth-child(even) td {
-    background: 0% 0% no-repeat padding-box;
-    background-color: rgba(198, 198, 198, 0.1);
```

Comment:
Rather than background every cell, I just background the row. This change made it easier to add space between rows and retain the background within that space.